go/chat/sender.go

```diff
@@ -1153,6 +1153,10 @@ func (s *BlockingSender) Send(ctx context.Context, convID chat1.ConversationID,
 	s.Debug(ctx, "Send: uid: %s in conversation %s (tlfName: %s) with status: %v", sender,
 		conv.GetConvID(), conv.Info.TlfName, conv.ReaderInfo.Status)
 
+	// Add txIDs to client header, if relevant
+	bodyTxIDs := utils.GetMessageBodyTxIDs(msg.ClientHeader.MessageType, msg.MessageBody)
+	msg.ClientHeader.TxIDs = &bodyTxIDs
```

Comment:
hoping that all messages go through this blocking Send method and the nonblocking Send method, so that the TxIDs always get added

Reply:
can you move this to the `Prepare`method?